signing_service/signing_service/signing_service.py

```diff
@@ -570,6 +583,15 @@ def _parse_arguments() -> argparse.Namespace:
     )
     crash_logger.handlers[0].client = raven_client  # type: ignore
 
+    if any('smtp-notifier' in args for args in argparse.ArgumentParser().parse_known_args()):  # type: ignore
```

Comment:
it is overcomplicated, you culd simply:
```suggestion
    if 'smtp-notifier' in args:
```

Reply:
I can't because it wouldn't work. I've been trying to explain it before. `subarsers` are not arguments, so `smtp-notifier` is never in `args`.

Reply:
then just:
```suggestion
   if hasattr(args, "from_email_address"): 
```